horizons/network/common.py

```diff
@@ -145,7 +145,7 @@ def __init__(self, state=Open):
 
 		def __str__(self):
 			strvals = ["Open", "Prepare", "Running"]
-			return "%s" % (strvals[self.state])
+			return "{}".format(strvals[self.state])
```

Comment:
Please change that to `return strvals[self.state]`.

It's not related to your changes, but it's a simple thing to do and we might forget about this later.